scrape_repo_information.py

```diff
@@ -0,0 +1,82 @@
+import urllib.request
+import pdb
+import xml.etree.ElementTree as xml
+import argparse, urllib.request
+import json
+import collections
+import time
+import urllib
+import os
+"""
+Script to scrape text from GitHub and into JSON format for all language pairs in Apertium.
+Stem counter by sushain
+"""
+
+CLIENT_ID = os.environ['GITHUB_CLIENT_ID']
+CLIENT_SECRET = os.environ['GITHUB_SECRET_CLIENT_ID']
+
+pairs = []
+Pair = collections.namedtuple('Pair', 'lg1 lg2 last_updated created direction repo stems')
+types = ["trunk", "nursery", "incubator", "staging"]
+
+params = urllib.parse.urlencode(dict({'client_id': CLIENT_ID, 'client_secret': CLIENT_SECRET}))
+
+def print_info(uri):
+    returned = get_info(uri)
+    return returned
+
+def get_info(uri):
+    try:
+        dictX = str((urllib.request.urlopen(uri)).read(), 'utf-8')
+        tree = xml.fromstring(dictX)
+    except:
+        return -1 # FIXME: error handling
+
+    return len(tree.findall("*[@id='main']/e//l"))
+
+if __name__ == "__main__":
+    for x in types:
```

Comment:
We're still not there. Everything here should be in a function called `main` to prevent global scope pollution and by convention.